src/preprocessing/removeNoise.jl

```diff
@@ -0,0 +1,24 @@
+function remove_noise(pipe::NlpPipe)::NlpPipe
+   patterns = [
+      r"<[^>]*>",                                        # HTML tags (very basic, just remove everything between < and >) TODO: Improve this
+      r"https?:\/\/[^\s]+",                              # URLs
+      r"www\.[^\s]+",                                    # URLs (starting with www)
+      r"[A-Za-z]:\\[^\s]+|\/[^\s]+",                     # File paths
+      r"[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}", # Emails
+      r"#\w+",                                           # Hashtags
+      r"\b\d{1,2}[/-]\d{1,2}[/-]\d{2,4}\b",              # Dates (MM/DD/YYYY or DD-MM-YY)
+      r"\b\d{1,2}:\d{2}(?:AM|PM)?\b",                    # Times (e.g., 10:30 AM)
```

Comment:
regex a bit wild. we need to remove satzzeichenn OR add them as extra token or both.